net/nginx/Makefile

```diff
@@ -507,4 +507,4 @@ define Package/nginx-mod-luci-ssl/install
 	$(INSTALL_DIR) $(1)/usr/bin
 endef
 
-$(eval $(call BuildPackage,nginx-mod-luci-ssl))
+$(eval $(call BuildPackage,nginx-mod-luci-ssl))
```

Comment:
??

Reply:
Good question, I'll see about it in an hour.